.vscode/extensions.json

```diff
@@ -6,7 +6,7 @@
     "dbaeumer.vscode-eslint",
     "esbenp.prettier-vscode",
     "voorjaar.windicss-intellisense",
-    "johnsoncodehk.volar",
+    "Vue.volar",
```

Comment:
codium does only know johnsoncodehk.volar ...

Reply:
https://open-vsx.org/extension/johnsoncodehk/volar

Reply:
But it contains very old version (0.30).

https://github.com/johnsoncodehk/volar - starting from 0.39 version it's published under Vue.volar name:
![attels](https://user-images.githubusercontent.com/165205/183896147-1bc147fc-6720-4b6d-a811-0d8a2ce1abeb.png)

Reply:
Also there is already fix for publishing in open-vsx 10 days ago, so hopefully it will be also soon there with correct name:
https://github.com/open-vsx/publish-extensions/commit/b4d5f50237fcce8007b26dd12cbcb2a4b8785e22